pii_scan.py

```diff
@@ -44,6 +44,13 @@ def analyze_text(text: str, show_supported=False, show_details=False, score_thre
                                 score=0.9)
     banner_id_recognizer = PatternRecognizer(supported_entity="BANNER_ID",
                                              patterns=[banner_id_pattern])
+    
+    zipcode_pattern = Pattern(name='zipcode_pattern',
```

Comment:
I think you have an extra space in your regex - should be no space between the | and the (
`(\b\d{5}(?!-)\b)|(\b\d{5}-\d{4}\b)`

If you are going to try to catch extended zip codes, ie 12345-1234 you need to include a test for that in your test case as well

Reply:
The Changes have been updated. Thanks